lib/bindConnectors/bindDynamicOutput.js

```diff
@@ -0,0 +1,31 @@
+var _                = require('lodash');
+var when             = require('when');
+var GenerateSchema = require('generate-schema');
+
+module.exports = function (message, options) {
+
+    return function (event) {
+
+        return when.promise(function (resolve, reject) {
+
+            when(message.dynamicOutput(event.body.input))
```

Comment:
Mmmmh I'm using Postman and try to send:
```
{
  "id": "123-def",
  "header": {
    "message": "list_records_dynamic_output"
  },
  "body": {
    "api_key": "keyGd4sUAwrALw0Xo",
    "base": "appBc3pXgbxSOjYQy",
	"table": "tbl08gffbF65BZpty"
  }
}
```
and I'm getting `undefined` for the `params`.